Pipfile

```
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
 "rfc5424-logging-handler" = "*"
-argo-workflows = ">=3.5"
+argo-workflows = ">=3.5<4"
```

we need a  
```suggestion
argo-workflows = ">=3.5,<4"
```
to have a version before 4. isn't @fridex ?

---

Yes, good catch :+1: